Added automatic grid detection to multi (#911)
danforthcenter/plantcv: docs/objects.md

## class Objects

A PlantCV data object class.

*class* plantcv.**Objects**

> `Objects` is a class that is used when plantcv functions draw curves on images. 

Reviewer: ```suggestion
`Objects` is a class that is used to manage image contours/objects and their hierarchical relationships. 
```

> `Objects` is a class that is used when plantcv functions draw curves on images. 

Reviewer: ```suggestion
`Objects` is a class that is used to manage image contours/objects and their hierarchical relationships. 
```

These attributes are used internally by PlantCV functions but also can be utilized by users. 

### Attributes

Attributes are accessed as Objects.*attribute*.

> **contours**: A list of all the points that form a curve. Based on [OpenCV contours](https://docs.opencv.org/3.4/d4/d73/tutorial_py_contours_begin.html)

Reviewer: ```suggestion
**contours**: A list of all the points that form the outline of a shape. Based on [OpenCV contours](https://docs.opencv.org/3.4/d4/d73/tutorial_py_contours_begin.html)
```

> **contours**: A list of all the points that form the outline of a shape. Based on [OpenCV contours](https://docs.opencv.org/3.4/d4/d73/tutorial_py_contours_begin.html)

Reviewer: ```suggestion
**contours**: A list contours (all the points that form the outline of a shape. Based on [OpenCV contours](https://docs.opencv.org/3.4/d4/d73/tutorial_py_contours_begin.html))
```

> **hierarchy**: An array that contains the hierarchy for each contour. Based on [OpenCV hierarchies](https://docs.opencv.org/4.x/d9/d8b/tutorial_py_contours_hierarchy.html)

Reviewer: ```suggestion
**hierarchy**: An array that contains the relationship between contours. Based on [OpenCV hierarchies](https://docs.opencv.org/4.x/d9/d8b/tutorial_py_contours_hierarchy.html)
```

> **hierarchy**: An array that contains the relationship between contours. Based on [OpenCV hierarchies](https://docs.opencv.org/4.x/d9/d8b/tutorial_py_contours_hierarchy.html)

Reviewer: ```suggestion
**hierarchy**: A list of hierarchies (An array that contains the relationship between contours. Based on [OpenCV hierarchies](https://docs.opencv.org/4.x/d9/d8b/tutorial_py_contours_hierarchy.html))
```

### Methods

> **save**(self, *filename*): Save results to a file.

Reviewer: ```suggestion
**save**(self, *filename*): Save objects to a file.
```

> **load**(*filename*): Load results from a file.

Reviewer: ```suggestion
**load**(*filename*): Load objects from a file.
```
